data/frontier jobs.txt

```diff
@@ -0,0 +1,156 @@
+# Copyright (c) 2017 by Michael Zahniser and Nate Graham
```

Comment:
I would do

```
# Copyright (c) 2017 by Michael Zahniser
# Copyright (c) 2018 by Nate Graham
```
Though we haven't ever really kept up with the years, and IANAL :)